OrbitService/LinuxTracingGrpcHandler.cpp

```diff
@@ -158,8 +158,15 @@ uint64_t LinuxTracingGrpcHandler::InternStringIfNecessaryAndGetKey(
 }
 
 void LinuxTracingGrpcHandler::SenderThread() {
+  constexpr std::chrono::duration kSendEvery = std::chrono::milliseconds{20};
```

Comment:
I wonder if this could be better if this was something like wait for X events or Y ms.

Reply:
I'm not sure how much better this is but I added a commit for this, you can have a look.